cirq/google/calibration/__init__.py

```diff
@@ -43,3 +44,14 @@
     run_zeta_chi_gamma_compensation_for_moments,
     try_convert_sqrt_iswap_to_fsim,
 )
+
+# pylint: disable=wrong-import-order
+from typing import Dict, Tuple
+
+import sys as _sys
+from cirq._compat import deprecate_attributes as _deprecate_attributes
+
+deprecated_constants: Dict[str, Tuple[str, str]] = {
+    'SQRT_ISWAP_PARAMETERS': ('v0.11', 'Use cirq.google.SQRT_ISWAP_INV_PARAMETERS instead'),
```

Comment:
`v0.11` should be `v0.12` - all deprecations must go through one release in the `deprecated` state before being fully removed.

Same in the file comment.

Reply:
Done